source/js/dg_maps.bindings.js

```diff
@@ -0,0 +1,45 @@
+/* global allieTabs */
+
+/**
+ * @file
+ * Filter component binding.
+ */
+(function (Drupal) {
+  'use strict';
+
+  Drupal.behaviors.gentBaseLoadDGMaps = {
+    attach: function (context, settings) {
+      if (!allieTabs) { // eslint-disable-line no-undef
+        return;
+      }
+
+      var selected = document.querySelectorAll('.js-map-tabs');
+      for (var s = selected.length; s--;) {
```

Comment:
A lot of nesting going on here...
Better to put sub-nesting in separate functions.